components/cu_transform/examples/basic_usage.rs

```diff
@@ -0,0 +1,53 @@
+use cu29::clock::CuDuration;
+use cu_spatial_payloads::Transform3D;
+use cu_transform::{StampedTransform, TransformTree};
+
+fn main() {
+    let mut tree = TransformTree::<f32>::new();
+
+    let world_to_robot = StampedTransform {
+        transform: {
+            let mut t = Transform3D::default();
+            t.mat[0][3] = 1.0; // X translation
+            t.mat[1][3] = 2.0; // Y translation
+            t
+        },
+        stamp: CuDuration(1000),
+        parent_frame: "world".to_string(),
+        child_frame: "robot".to_string(),
+    };
+
+    let robot_to_camera = StampedTransform {
+        transform: {
+            let mut t = Transform3D::default();
+            t.mat[0][3] = 0.2; // X translation
+            t.mat[1][3] = 0.0; // Y translation
+            t.mat[2][3] = 0.3; // Z translation
+            t
+        },
+        stamp: CuDuration(1000),
+        parent_frame: "robot".to_string(),
```

Comment:
could we use anything that is stringable?

for example the user could have a clean strong typed enum and they would implement Display for debugging purposes.
If the user doesn't want to bother, they could use a Smallstr (a la monitoring UI?)